core/src/server/helpers.rs

```diff
@@ -119,8 +119,8 @@ impl MethodSink {
 				tracing::error!("Error serializing response: {:?}", err);
 
 				if err.is_io() {
-					let data = to_json_raw_value(&format!("Exceeded max limit {}", self.max_response_size)).ok();
-					let err = ErrorObject::borrowed(OVERSIZED_RESPONSE_CODE, &OVERSIZED_RESPONSE_MSG, data.as_deref());
+					let data = format!("Exceeded max limit {}", self.max_response_size);
```

Comment:
makes more sense to use `ErrrorObject::owned` when we already have to allocate a `String` here.